api/common/unstructured.go

```diff
@@ -0,0 +1,56 @@
+package common
+
+import (
+	encodingjson "encoding/json"
+	"fmt"
+)
+
+// Minimal unstructured type to add in Kubernetes types.
+type Unstructured map[string]interface{}
```

Comment:
It seems `Unstructured` didn't be used in this PR.

Reply:
Right.